package.json

```diff
@@ -28,9 +28,6 @@
     "webpack-cli": "3.3.10"
   },
   "browserslist": [
-    "> 1%",
-    "last 2 firefox versions",
-    "last 2 safari versions",
-    "ie 11"
+    "defaults"
```

Comment:
So what `defaults` then?

Reply:
See first post. It is `0.5%, last 2 versions, Firefox ESR, not dead` which resolves to those browsers listed there.